sermant-plugins/sermant-mq-grayscale/config/config.yaml

```diff
@@ -0,0 +1,25 @@
+# configuration of rocketMq message grayscale
```

Comment:
I think this plugin is not just about rocketmq in long term

Reply:
fixed